substrate/primitives/core/src/const_hex2array.rs

```diff
@@ -112,4 +122,28 @@ mod testh2b {
 			]
 		);
 	}
+
+	#[test]
+	#[should_panic]
```

Comment:
```suggestion
	#[should_panic = "hex string length is not valid"]
```
(same below)

Also why not just:

```
	#[test]
	#[should_panic = "hex string length is not valid"]
	fn t_panic_incorrect_length() {
		let _ = hex2array::<2>("454");
	}
```

Reply:
yeah, thanks, I over-complicated that.

Reply:
f01fe2f